idf_baseline/qa-data-only-idf.py

```diff
@@ -1,40 +1,96 @@
 import argparse
 import os
+import sys
+import re
 import numpy as np
 from collections import defaultdict
 import string
+import subprocess
+import shlex
 
 import nltk
-nltk.download('stopwords')
+nltk.download('stopwords', quiet=True)
```

Comment:
Still gives a download error if not connected to the internet. However, the program proceeds correctly of the stopwords exist locally (have been pre-downloaded or pre-installed).